server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/PLController.java

```diff
@@ -58,13 +61,15 @@ public SuccessResponse<String> startBatchCompile(@PathVariable String sessionId,
     @ApiOperation(value = "end batchCompile", notes = "终止批量编译")
     @RequestMapping(value = {"/{sessionId}/databases/{databaseName}/batchCompilations/{id}",
             "/{sessionId}/currentDatabase/batchCompilations/{id}"}, method = RequestMethod.DELETE)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sessionId")
```

Comment:
In fact, the state-route of the `id` is dependent on the `sessionId`, because the request that creates the `id` uses the `stateId` of `sessionId`; therefore, it can be considered that the state-route of the`id` is same as  the `sessionId`'s.